modules/api/src/main/java/org/apache/ignite/schema/definition/builder/TableDefinitionBuilder.java

```diff
@@ -27,6 +27,14 @@
  * Table definition builder.
  */
 public interface TableDefinitionBuilder extends SchemaObjectBuilder {
+    /**
+     * Adds columns to the table.
+     *
+     * @param columns Table columns definitions.
+     * @return {@code This} for chaining.
+     */
+    TableDefinitionBuilder columns(Iterable<ColumnDefinition> columns);
```

Comment:
Columns collection must be ordered. This is described in the SQL standard.
Iterable implies no specific order.

Reply:
got it !